### PR TITLE
docs: revise structure

### PR DESCRIPTION
**Pull request by ltalirz**
* move short introduction to docs landing page
 * move installation information from README to docs; clarify difference between app and package
 * adapt "AiiDAlab widgets" name https://github.com/aiidalab/aiidalab-widgets-base/issues/226

### CODE REVIEW

**Comment by ltalirz** [comment]
@csadorf might be worth having a look at this PR as well.
I still plan to add some minor changes here; the actual docs on the individual widgets will follow in a separate PR

will now be out for a couple of hours

File: docs/source/installation.rst [hunk 1]
```diff
@@ -0,0 +1,32 @@
+************
+Installation
+************
+
+The AiiDAlab widgets app comes pre-installed in the AiiDAlab environment.
+
+The app contains
+ * the `aiidalab-widgets-base` python package that contains the widgets (`from aiidalab_widgets_base import ...`)
```
Comment (csadorf) [inline docs/source/installation.rst:8]:
```suggestion
 * the `aiidalab-widgets-base` Python package that contains the widgets (`from aiidalab_widgets_base import ...`)
```
The name of the "Python" programming language should be capitalized, see for example: https://en.wikipedia.org/wiki/Python_(programming_language) .

File: docs/source/widget-list/index.rst [hunk 2]
```diff
@@ -4,6 +4,12 @@ List of widgets
 
 Here we collect the information about all the widgets available in AWB and provide a simple example of how to use them.
 
+
+Using the widgets usually just involves importing and displaying them.
+For demos, have a look at the jupyter notebooks (`.ipynb` extension) in
+this folder.
```
Comment (csadorf) [inline docs/source/widget-list/index.rst:10]:
```suggestion
For demos, have a look at the jupyter notebooks (`.ipynb` extension) in this folder.
```
We should probably install a linter, but generally we should follow the "one sentence per line" rule.

File: README.md [hunk 3]
```diff
@@ -1,30 +1,19 @@
 [![Documentation Status](https://readthedocs.org/projects/aiidalab-widgets-base/badge/?version=latest)](https://aiidalab-widgets-base.readthedocs.io/en/latest/?badge=latest)
 
-# Base widgets for AiiDAlab
+# AiiDAlab Widgets
```
Comment (csadorf) [inline README.md:3]: I am open to renaming the package, but obviously this **must** be discussed and agreed upon within the team.

File: README.md [hunk 4]
```diff
@@ -1,30 +1,19 @@
...
+[AiiDAlab](https://materialscloud.org/aiidalab) applications typically involve some of following steps:
 
-Install the `aiidalab_widgets_base` python package via:
-```
-pip install aiidalab-widgets-base
-```
+ * prepare the input for a calculation (e.g. an atomic structure)
+ * select computational resources and submit a calculation to AiiDA
+ * monitor a running calculation
+ * find and analyze the results of a calculation
```
Comment (csadorf) [inline README.md:10]: Those are all sentences as far as I can tell.
```suggestion
 * Prepare the input for a calculation (e.g. an atomic structure).
 * Select computational resources and submit a calculation to AiiDA.
 * Monitor a running calculation.
 * Find and analyze the results of a calculation.
```
Same below.

File: docs/source/contribute/index.rst [hunk 5]
```diff
@@ -1,10 +1,10 @@
...
 
-* `Create issues <https://github.com/aiidalab/aiidalab-widgets-base/issues/new>`_ in case you found a bug or you need a feature that is not yet implemented.
-* Contribute new widgets.
-* Help us with fixing the `existing issues <https://github.com/aiidalab/aiidalab-widgets-base/issues>`_.
-* Extend the documentation.
\ No newline at end of file
+* Contribute a widget you created that would be of general interest to AiiDAlab users
+* `Report bugs <https://github.com/aiidalab/aiidalab-widgets-base/issues>`_
+* `Feature requests <https://github.com/aiidalab/aiidalab-widgets-base/issues>`_
+* Help us improve the documentation of widgets
```
Comment (csadorf) [inline docs/source/contribute/index.rst:10]: These are sentences.

File: docs/source/index.rst [hunk 6]
```diff
@@ -1,29 +1,26 @@
...
+Welcome to the AiiDAlab Widgets documentation!
+##############################################
 
+`AiiDAlab <https://materialscloud.org/aiidalab>`_ applications typically involve some of following steps:
 
-AiiDAlab Widgets Base (AWB) is a set of handy widgets that are used in AiiDAlab to create applications.
+ * prepare the input for a calculation (e.g. an atomic structure)
+ * select computational resources and submit a calculation to AiiDA
+ * monitor a running calculation
+ * find and analyze the results of a calculation
```
Comment (csadorf) [inline docs/source/index.rst:10]: These are sentences.

**Comment by ltalirz** [comment]
@csadorf Thanks for the first check; from my side this is now ready to be merged.

I switched the layout to the pydata-sphinx-theme (which we use also for aiida-core), since the bulleted lists were displayed in a weird way in the old theme.
I've built it locally and to me the docs look fine.

File: docs/source/widget-list/index.rst [hunk 7]
```diff
@@ -4,10 +4,8 @@ List of widgets
 
 This is a collection of Jupyter notebooks that illustrate how to use the widgets available from this library.
 
-
-Using the widgets usually just involves importing and displaying them.
-For demos, have a look at the jupyter notebooks (`.ipynb` extension) in this folder.
-
+In the AiiDAlab, these notebooks can be found in the ``notebooks`` directory of the "AiiDAlab Widgets" app.
```
Comment (csadorf) [inline docs/source/widget-list/index.rst:7]:
```suggestion
In AiiDAlab, these notebooks can be found in the ``notebooks`` directory of the "AiiDAlab Widgets" app.
```

**Review by csadorf — DISMISSED** [review]
One suggestion, otherwise good! 👍

**Pull request merged**